src/main/java/com/magnaboy/CitizenRegion.java

```diff
@@ -196,6 +206,25 @@ public static void spawnScenery(SceneryInfo info) {
 		dirtyRegion(region);
 		plugin.refreshEntityCollection();
 		plugin.updateAll();
+		return scenery;
+	}
+
+	public static void saveEntity(EntityInfo info) {
+		if (info.entityType == EntityType.Scenery) {
+			//TODO
```

Comment:
Intentionally left todo?

Reply:
Yes, scenery can't be selected yet, so its irrelevant at the moment.